src/core_atmosphere/CMakeLists.txt

```diff
@@ -75,6 +74,18 @@ set(_atm_core_physics_wrf_srcs
         module_sf_urban.F)
 list(TRANSFORM _atm_core_physics_wrf_srcs PREPEND physics/physics_wrf/)
 
+# chemistry/
+set(_atm_core_chemistry_srcs
+        chemistry_opp/o3_gfs.F
+        mpas_chemistry_driver.F
+        mpas_chemistry_init.F
+        mpas_chemistry_interface.F
+        mpas_chemistry_manager.F
+        mpas_chemistry_opp.F
+        mpas_chemistry_todynamics.F)
```

Comment:
@metdyn Now I see there are 9 added files in Laura's branch. Here you only have 7 files. I guess this is is the cause of the build problem.

Reply:
Looks like missing
A src/core_atmosphere/chemistry/mpas_chemistry_packages.F

Reply:
I am checking now.